bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/Ghost.scala

```diff
@@ -320,44 +320,52 @@ object Ghost {
   val EXPOSURE_TIME_RED_KEY = new ItemKey(INSTRUMENT_KEY, EXPOSURE_TIME_RED_PROP)
   val EXPOSURE_TIME_BLUE_PROP = "blueExposureTime"
+  val COUNT_BLUE = "blueExposureCount"
+  val BINNING_BLUE = "blueBinning"
+  val NOISE_GAIN_BLUE = "blueReadNoiseGain"
   val DEF_EXPOSURE_TIME_BLUE = 10.0
   val EXPOSURE_TIME_BLUE_KEY = new ItemKey(INSTRUMENT_KEY, EXPOSURE_TIME_BLUE_PROP)
 
   // The names of the base position / IFUs.
-  val BaseRADegrees: String  = "baseRADeg"
-  val BaseRAHMS: String      = "baseRAHMS"
-  val BaseDecDegrees: String = "baseDecDeg"
-  val BaseDecDMS: String     = "baseDecDMS"
-
-  val SRIFU1Name: String     = "srifu1Name"
-  val SRIFU1RADeg: String    = "srifu1CoordsRADeg"
-  val SRIFU1DecDeg: String   = "srifu1CoordsDecDeg"
-  val SRIFU1RAHMS: String    = "srifu1CoordsRAHMS"
-  val SRIFU1DecDMS: String   = "srifu1CoordsDecDMS"
-  val SRIFU1Guiding: String  = "srifu1Guiding"
-
-  val SRIFU2Name: String     = "srifu2Name"
-  val SRIFU2RADeg: String    = "srifu2CoordsRADeg"
-  val SRIFU2DecDeg: String   = "srifu2CoordsDecDeg"
-  val SRIFU2RAHMS: String    = "srifu2CoordsRAHMS"
-  val SRIFU2DecDMS: String   = "srifu2CoordsDecDMS"
-  val SRIFU2Guiding: String  = "srifu2Guiding"
-
-  val HRIFU1Name: String     = "hrifu1Name"
-  val HRIFU1RADeg: String    = "hrifu1CoordsRADeg"
-  val HRIFU1DecDeg: String   = "hrifu1CoordsDecDeg"
-  val HRIFU1RAHMS: String    = "hrifu1CoordsRAHMS"
-  val HRIFU1DecDMS: String   = "hrifu1CoordsDecDMS"
-  val HRIFU1Guiding: String  = "hrifu1Guiding"
-
-  val HRIFU2Name: String     = "hrifu2Name"
-  val HRIFU2RADeg: String    = "hrifu2CoordsRADeg"
-  val HRIFU2DecDeg: String   = "hrifu2CoordsDecDeg"
-  val HRIFU2RAHMS: String    = "hrifu2CoordsRAHMS"
-  val HRIFU2DecDMS: String   = "hrifu2CoordsDecDMS"
+  val BASE_RA_DEGREES: String  = "baseRADeg"
+  val BASE_RA_HMS: String      = "baseRAHMS"
+  val BASE_DEC_DEGREES: String = "baseDecDeg"
+  val BASE_DEC_DMS: String     = "baseDecDMS"
+
+  val FIBER_AGITATOR: String   = "enableFiberAgitator"
+
+  val SRIFU1_NAME: String      = "srifu1Name"
+  val SRIFU1_RA_DEG: String    = "srifu1CoordsRADeg"
+  val SRIFU1_DEC_DEG: String   = "srifu1CoordsDecDeg"
+  val SRIFU1_RA_HMS: String    = "srifu1CoordsRAHMS"
+  val SRIFU1_DEC_DMS: String   = "srifu1CoordsDecDMS"
+  val SRIFU1_GUIDING: String   = "srifu1Guiding"
+
+  val SRIFU2_NAME: String      = "srifu2Name"
+  val SRIFU2_RA_DEG: String    = "srifu2CoordsRADeg"
+  val SRIFU2_DEC_DEG: String   = "srifu2CoordsDecDeg"
+  val SRIFU2_RA_HMS: String    = "srifu2CoordsRAHMS"
+  val SRIFU2_DEC_DMS: String   = "srifu2CoordsDecDMS"
+  val SRIFU2_GUIDING: String   = "srifu2Guiding"
+
+  val HRIFU1_NAME: String      = "hrifu1Name"
+  val HRIFU1_RA_DEG: String    = "hrifu1CoordsRADeg"
+  val HRIFU1_DEC_DEG: String   = "hrifu1CoordsDecDeg"
+  val HRIFU1_RA_HMS: String    = "hrifu1CoordsRAHMS"
+  val HRIFU1_DEC_DMS: String   = "hrifu1CoordsDecDMS"
+  val HRIFU1_GUIDING: String   = "hrifu1Guiding"
```

Comment:
@sraaphorst This doesn't seem to be in use

Reply:
I'll take a look.